.travis.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         - python usage_tests.py
         - python assumption_tests.py
         - python fulara_tests.py
+        - python fulara_usage_tests.py
```

Comment:
You can add your tests to the usage_tests file: just add them in the definition of `test_suite()` (see assumption_tests, for example).